src/client/components/app/LineDetailsDisplay/LineMetadata.tsx

```diff
@@ -383,31 +470,50 @@ export const LineMetadata = () => {
     return;
   };
 
-  console.log('Metadata staus', status);
   return (
     <>
       <ThreeColumnLayout>
         <MetadataBox>
-          <p>
+          <MetadataRow>
+            <Tooltip placement="bottom-start" tooltipComponent={<>{t('lineDetails:metadata.tooltip.status')}</>}>
+              <StyledIcon Component={InfoIcon} size="1.5rem" />
+            </Tooltip>
             <MetadataTitle>{t('lineDetails:metadata.status')}: </MetadataTitle>{' '}
             <StatusWithColor status={status}>{status[0].toUpperCase() + status.substring(1)}</StatusWithColor>{' '}
-          </p>
-          <p>
+          </MetadataRow>
+          <MetadataRow>
+            <Tooltip placement="bottom-start" tooltipComponent={<>{t('lineDetails:metadata.tooltip.start')}</>}>
+              <StyledIcon Component={InfoIcon} size="1.5rem" />
+            </Tooltip>
             <MetadataTitle>{t('lineDetails:metadata.start')}: </MetadataTitle> {startDateHumanized}
-          </p>
-          <p>
+          </MetadataRow>
+          <MetadataRow>
+            <Tooltip placement="bottom-start" tooltipComponent={<>{t('lineDetails:metadata.tooltip.end')}</>}>
+              <StyledIcon Component={InfoIcon} size="1.5rem" />
+            </Tooltip>
             <MetadataTitle>{t('lineDetails:metadata.end')}: </MetadataTitle> {endDateHumanized}
-          </p>
-          <p>
+          </MetadataRow>
+          <MetadataRow>
+            <Tooltip
+              placement="bottom-start"
+              tooltipComponent={<>{t('lineDetails:metadata.tooltip.total-interest-paid')}</>}
+            >
+              <StyledIcon Component={InfoIcon} size="1.5rem" />
+            </Tooltip>
             <MetadataTitle>{t('lineDetails:metadata.total-interest-paid')}: </MetadataTitle> $
             {humanize('amount', totalInterestRepaid, 18, 2)}
```

Comment:
This is not quite right because we are normalizing all the stored `BigDecimal` values in state to 18 decimals so we don't misaggregate things (i.e. USDC and DAI). 

Regardless, I have replaced all hardcoded values of 18 with this from `format.ts`: 

```
export const BASE_DECIMALS = 18;
```